### ISSUE TITLE
when  i run './run.sh',about step 840 the loss begin to display NaN.

### ISSUE THREAD

**Issue by Yousefcheng** [laiguokun/bert-cloth#5]
this is log info:    step: 830 | train loss: nan | train acc 0.5820512771606445
out, _ = self.bert(articles, attention_mask = articles_mask,
            output_all_encoded_layers=False)

in this code, the result of out is [NAN,NAN,NAN..........]

**Comment by RainFZY** [comment]
> this is log info: step: 830 | train loss: nan | train acc 0.5820512771606445
> out, _ = self.bert(articles, attention_mask = articles_mask,
> output_all_encoded_layers=False)
> 
> in this code, the result of out is [NAN,NAN,NAN..........]

Initially I set CUDA_VISIBLE_DEVICES to 0,1,2,3,4,5,6,7 and meet the same problem, then I cut the number down to 0,1 and solve the problem.

**Comment by birdmanmandbir** [comment]
> > this is log info: step: 830 | train loss: nan | train acc 0.5820512771606445
> > out, _ = self.bert(articles, attention_mask = articles_mask,
> > output_all_encoded_layers=False)
> > in this code, the result of out is [NAN,NAN,NAN..........]
> 
> Initially I set CUDA_VISIBLE_DEVICES to 0,1,2,3,4,5,6,7 and meet the same problem, then I cut the number down to 0,1 and solve the problem.

But if I ser the number to 0,1, the memory is not enough.

Besides, could you tell me the versions of python and pytorch you used?
